src/projection/node/create-projection-node.ts

```diff
@@ -432,22 +446,26 @@ export function createProjectionNode<I>({
 
         // only on the root
         blockUpdate() {
-            this.updateBlocked = true
+            this.updateManuallyBlocked = true
         }
 
         unblockUpdate() {
-            this.updateBlocked = false
+            this.updateManuallyBlocked = false
+        }
+
+        isUpdateBlocked() {
+            return this.updateManuallyBlocked || this.updateBlockedByResize
         }
 
         // Note: currently only running on root node
         startUpdate() {
-            if (this.updateBlocked) return
+            if (this.updateManuallyBlocked) return
```

Comment:
Shouldn't we check `isUpdateBlocked()` here?

Reply:
Great catch thanks

Reply:
Maybe also create a test around it?